.circleci/config/jobs/test-api.yml

```diff
@@ -0,0 +1,13 @@
+executor: go-machine-medium
+steps:
+- checkout
+- install-go
+- run:
+    name: "Run API Tests"
+    no_output_timeout: 15m
+    command: |
+      make test-api
+- slack/status:
+    fail_only: true
+    only_for_branches: master
```

Comment:
main branch?

Reply:
good catch, and actually its set to `master` for the [current ci job](https://github.com/hashicorp/boundary/blob/739222d06e3a812c44eb64494ad21cd39d54dcf6/.circleci/config/jobs/build.yml#L21-L24) which probably means it isn't working at all...I might just remove these slack status blocks for now and look into that some other time.

Reply:
I'm ready for  the proposed migration to GitHub actions.